clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -1407,14 +1407,61 @@ check_filter_endpoint()
     return true;
 }
 
+bool
+check_timestamps_increase_monotonically(void)
+{
+    // Positive test: timestamps increase monotonically.
+    {
+        std::vector<memref_t> memrefs = {
+            gen_marker(1, TRACE_MARKER_TYPE_TIMESTAMP, 0),
```

Comment:
We've also started using a local constant like `TID` or putting `/*tid=*/, ` to make it easier to understand what this 1 constant is.

Reply:
I've changed the code to use a constant TID.